### ISSUE TITLE
Is there any way to disable VLC output?

### ISSUE THREAD

**Issue by bbhxwl** [donandren/vlcsharpavalonia#17]
Is there any way to disable VLC output?

**Comment by 07artem132** [comment]
yes, check out the cli, it works as an option when creating an object.
https://wiki.videolan.org/VLC_command-line_help/

**Comment by bbhxwl** [comment]
![image](https://user-images.githubusercontent.com/26525599/129000178-9c4f6495-0f37-486b-a629-7f309efb6779.png)
I mean, does this output need to be disabled on the command line?

**Comment by 07artem132** [comment]
Console logger (console)
  -q, --quiet, --no-quiet        Be quiet
                                 (default disabled)
          Turn off all messages on the console.

**Comment by bbhxwl** [comment]
> Console logger (console)
> -q, --quiet, --no-quiet Be quiet
> (default disabled)
> Turn off all messages on the console.

I haven't set it. It's on by default? Can the code be set to close?

**Comment by bbhxwl** [comment]
vlc -q, --quiet, --no-quiet     ？

**Comment by bbhxwl** [comment]
My English is not good. I don't know where you wrote this?

**Comment by 07artem132** [comment]
Try to create an object like this: new LibVLC("--quiet")

**Comment by bbhxwl** [comment]
> Try to create an object like this: new LibVLC("--quiet")

Thank you. It's really useful.

![image](https://user-images.githubusercontent.com/26525599/129320665-8f2d341b-5b4e-4ae0-a260-b08ca0ad5502.png)

**Comment by bbhxwl** [comment]
> Try to create an object like this: new LibVLC("--quiet")

![image](https://user-images.githubusercontent.com/26525599/129321801-95c690f8-1ee1-438d-93e4-d53b5e0df631.png)
If the proportion of the video is poor, is there a way to fill it? Otherwise there will be a black border
